index.js

```diff
@@ -1,20 +1,342 @@
+  {
+    titleTop: 'Professional art',
+    titleBottom: 'printing data',
+    description:
+      'A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry`s standard',
+    image: './img/art1.png',
+    technologies: ['css', 'html', 'bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    titleTop: 'Professional art',
+    titleBottom: 'printing data',
+    description:
+      'A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry`s standard',
+    image: './img/art1.png',
+    technologies: ['css', 'html', 'bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    titleTop: 'Professional art',
+    titleBottom: 'printing data',
+    description:
+      'A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry`s standard',
+    image: './img/art1.png',
+    technologies: ['css', 'html', 'bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    titleTop: 'Professional art',
+    titleBottom: 'printing data',
+    description:
+      'A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry`s standard',
+    image: './img/art1.png',
+    technologies: ['css', 'html', 'bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    titleTop: 'Professional art',
+    titleBottom: 'printing data',
+    description:
+      'A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry`s standard',
+    image: './img/art1.png',
+    technologies: ['css', 'html', 'bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+  {
+    titleTop: 'Professional art',
+    titleBottom: 'printing data',
+    description:
+      'A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry`s standard',
+    image: './img/art1.png',
+    technologies: ['css', 'html', 'bootstrap'],
+    seeLive: '#',
+    seeSource: '#',
+  },
+];
+
+let selectedProject;
+
+const menuIcon = document.querySelector('#menu-icon');
+const menuToggle = document.querySelector('#menu-toggle');
+const menuSection = document.querySelector('#menu-section');
+
+const removePopup = () => {
+  document.body.style.overflowY = 'unset';
+  const popupContainer = document.getElementById('popup-container');
+  document.querySelector('.about-overly').style.display = 'block';
+  selectedProject = null;
+  popupContainer.remove();
+};
+
+const addPopup = () => {
+  if (selectedProject) {
+    const popupContainer = document.createElement('div');
+    popupContainer.setAttribute('id', 'popup-container');
+    document.body.style.overflowY = 'hidden';
+    document.querySelector('.about-overly').style.display = 'none';
+    document.body.appendChild(popupContainer);
+    const popup = document.createElement('div');
+    popup.setAttribute('class', 'popup');
```

Comment:
- [ ] As mentioned by the previous reviewer, the popup is not properly displaying. Currently it is not scrollable to enable users to see the text and button hidden down.

![image](https://user-images.githubusercontent.com/84156714/218283693-16dd6cd7-cf10-4c12-aea3-fdac3bd0b85c.png)

I suggest you add the overflow- [See tutorial](https://www.w3schools.com/css/css_overflow.asp) on how to do this.

The desktop view hides the popup heading
![image](https://user-images.githubusercontent.com/84156714/218283746-caebe567-d5dc-49fe-95bb-f558965ceafc.png)


Please fix this